app/autotranslate/client/views/autoTranslateFlexTab.js

```diff
@@ -99,7 +99,7 @@ Template.autoTranslateFlexTab.onCreated(function() {
 	this.saveSetting = () => {
 		const field = this.editing.get();
 		const subscription = Subscriptions.findOne({ rid: this.rid, 'u._id': Meteor.userId() });
-		const previousLanguage = subscription.autoTranslateLanguage;
+		const previousLanguage = subscription && subscription.autoTranslateLanguage;
```

Comment:
How could you toggle language without being subscribed to the room?

Reply:
You can't, currently. That would be another fix we should do, or at least hide the translation panel / button if not subscribed.

Reply:
ok, doesn't hurt anyway